src/app/components/question-approval-page/question-approval-page.component.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { QuestionServiceStub } from 'src/testing/question.service.stub';
+import { QuestionApprovalPageComponent } from './question-approval-page.component';
+import { ClarityModule } from '@clr/angular';
+import { ReactiveFormsModule } from '@angular/forms';
+import { QuestionService } from 'src/app/services/question.service';
+import { KatexModule } from 'ng-katex';
+
+// describe('QuestionApprovalPageComponent', () => {
```

Comment:
Let's avoid doing this. As a minimum each component should have the `it should create` unit tests. Reason being if we change another part of the codebase, we don't want to suddenly find out we've broken this component.

Reply:
Also we've got 3 methods within the approval component, lets get those unit tested as well.